modules/twinkleprod.js

```diff
@@ -181,24 +181,24 @@ Twinkle.prod.callbacks = {
 		var statelem = apiobj.statelem;
 		var params = apiobj.params;
 
-		// Check talk page for templates indicating prior XfD
+		// Check talk page for templates indicating prior XfD or PROD
 		if ($(xmlDoc).find('templates').length) {
-			statelem.warn('Previous XfD template found on talk page, aborting procedure');
-			return;
-		}
-
-		// Check talk page for category indicating prior PROD
-		if ($(xmlDoc).find('categories').length) {
-			if (params.blp) {
-				if (!confirm('Previous PROD nomination found on talk page. Do you still want to continue applying BLPPROD? ')) {
-					statelem.warn('Previous PROD found on talk page, aborted by user');
+			var template = $(xmlDoc).find('templates tl')[0].getAttribute('title');
```

Comment:
Is the return order guaranteed here?  I seem to always get `Old XfD multi` before `Old prod`, and I think it sorts capitalized templates first, so if true, `Old prod` should actually *always* be last given our list of templates.  This works because of that, but at the very least should include a comment as a warning either here or below in case anything changes.

That being said, wouldn't it be better to check the length of `$(xmlDoc).find('templates tl')`?  That way, we show the PROD message iff there's one template and it's Old prod; for >1 templates or just one but *not* Old prod, show the XfD message.  That should cover our bases more thoroughly.

Reply:
> I think that's because it sorts capitalized templates first. 

Indeed, uppercase letters have lesser ASCII values are thus sorted ahead.

Regarding the 2nd para - yes, that sounds more robust. Changed.